konnectivity-client/go.mod

```diff
@@ -5,18 +5,17 @@ go 1.17
 // Prefer to keep requirements compatible with the oldest supported
 // k/k minor version, to prevent client backport issues.
 require (
-	github.com/golang/protobuf v1.4.3
 	go.uber.org/goleak v1.2.0
-	google.golang.org/grpc v1.27.1
+	google.golang.org/grpc v1.48.0
```

Comment:
k/k release-1.24 has google.golang.org/grpc v1.40.0. Don't we want to stay at or below that here?

Reply:
+1. Good catch.

I did a 'go install google.golang.org/grpc@v1.40.0' Need to track down where the 1.48.0 came from. (?bad rebase?)

Reply:
Fixed. Looks like it's 'go mod tidy' which is flipping it to 1.48.0.